Kickstarter-iOS/Features/PledgeOverTime/Views/Cells/PledgePaymentPlanCell.swift

```diff
@@ -0,0 +1,171 @@
+import Library
+import UIKit
+
+final class PledgePaymentPlanCell: UITableViewCell, ValueCell {
+  // MARK: properties
+
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+
+  private lazy var leftColumnStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var rigthColumnStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var titleLabel = { UILabel(frame: .zero) }()
+  private lazy var subtitleLabel = { UILabel(frame: .zero) }()
+  private lazy var checkmarkImageView: UIImageView = { UIImageView(frame: .zero) }()
```

Comment:
nit: can we rename this? Maybe just to `imageView`? This might just be my opinion, so you can ignore it, but I was pretty surprised based on these names when I realized that this image could also represent the unselected state.

Reply:
thank you for pointing this out! I’ve decided to go with `selectionIndicatorImageView` as the new name, since it clearly reflects the purpose of this image view: indicating whether the option is selected or not. I think this makes the code more descriptive and avoids potential confusion in the future. Let me know if you have any further thoughts or suggestions! 😊